src/edge_playback/__main__.py

```diff
@@ -46,18 +68,30 @@ def _main() -> None:
                 f"--write-media={mp3_fname}",
                 f"--write-subtitles={srt_fname}",
             ]
-            + sys.argv[1:]
+            + tts_args
         ) as process:
             process.communicate()
 
-        with subprocess.Popen(
-            [
-                "mpv",
-                f"--sub-file={srt_fname}",
-                mp3_fname,
-            ]
-        ) as process:
-            process.communicate()
+        if platform.system() == "Windows" and not use_mpv:
+
+            def mci_send(msg: str) -> None:
+                result = ctypes.windll.winmm.mciSendStringW(msg, 0, 0, 0)
+                if result != 0:
+                    print(f"Error {result} in mciSendString {msg}")
+
+            mci_send("Close All")
+            mci_send(f'Open "{mp3_fname}" Type MPEGVideo Alias theMP3')
```

Comment:
Technically if a user overrode the file path using EDGE_PLAYBACK_MP3_FILE and included a `'` this will break as it's not escaped. As it's a niche case I'm willing to merge this if escaping the file path is too difficult.

Reply:
I haven't actually been able to reproduce this with `'`, it seems to work ok with that as well as assorted unicode characters and very long filenames. Just to be on the safe side though I added a call to `_GetShortPathNameW` to get the DOS-style safe short path for the file. `"` does seem to break playback, but it also breaks `src/edge_tts/util.py:60`

Reply:
Sorry I might using a double quote not `'`. Typo.

Reply:
I looked it up and `"` is forbidden in filenames on Windows: https://en.wikipedia.org/wiki/Filename

Reply:
Nevermind I didn't know that Windows doesn't allow double quotes in filenames. https://stackoverflow.com/a/52618372